spec/presenters/thread_spec.rb

```diff
@@ -167,6 +167,15 @@ def random_flag_abuses!(comment)
         end
       end
 
+      it "handles merge_question_type_responses=true" do
+        @threads_with_num_comments.each do |thread, num_comments|
+          is_endorsed = num_comments > 0 && endorse_responses
+          hash = ThreadPresenter.new(thread, @reader, false, num_comments, is_endorsed, nil).to_hash(true, 0, default_resp_limit, true, false, true, true)
+          check_thread_result(@reader, thread, hash)
+          check_thread_response_paging(thread, hash, 0, default_resp_limit, false, false, true, true)
```

Comment:
this is the same check we have for before this new param was added!
Can we be more specific in the test to check `merge_question_type_responses` params impact on the results returned! Right now, we are sort of only checking if we get a response or not

Reply:
`check_thread_response_paging` is handling `merge_question_type_responses` impacts on the results returned